.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,20 @@
-- [ ] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
+## Check List
+
+**Please read and check followings before submitting a PR.**
+
+- [ ] I want to publish my theme on Hexo official website.
+    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
+    - [ ] `link` URL is correct.
+    - [ ] `preview` URL is correct.
+    - [ ] `preview` URL web site is rendered correctly.
+    - [ ] Add a screenshot to `source/themes/screenshots`.
+    - [ ] Screenshot file name with the same as `name`.
```

Comment:
"...filename is same as value of `name`..."

Reply:
done :)